.github/workflows/build-ffmpeg.yml

```diff
@@ -149,6 +149,13 @@ jobs:
           echo "root_path=$PWD" >> $GITHUB_OUTPUT
           mkdir -p ./bin ./ffmpeg_build ./ffmpeg_sources
 
+      - name: patch
+        shell: bash
+        working-directory: ffmpeg_sources/ffmpeg
+        run: |
+          git clean -f
+          git apply ../../ffmpeg_patches/ffmpeg/03-amfenc-disable-buffering-blocking-delay-in-IPP-mode.patch
```

Comment:
`git apply -v --ignore-whitespace ...` will work on Windows. I don't know why. I can apply the patches on my local windows machine just fine.